CHANGELOG.md

```diff
@@ -13,9 +14,11 @@ Two faster strategies have been added for this use case: `xxh64` and
 lightweight. Read more about it in the [call caching documentation](
 https://cromwell.readthedocs.io/en/stable/Configuring/#call-caching).
 
-### Bugfixes
+### Bug fixes
 
-+ Fixed a bug where the `use_relative_output_paths` option would not preserve intermediate folders.
+* Fixed a bug where the `size(...)` function did not work correctly on files from the input section
```

Comment:
```suggestion
* Fixed a bug where the `size(...)` function did not work correctly on shared filesystem files from the input section
```

Reply:
Yes, the shared filesystem should be explicitly mentioned. I have changed the sentence so it flows better and correctly mentions the shared filesystem.